src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTestFiller.yml

```diff
@@ -0,0 +1,64 @@
+doubleSelfdestructTest:
+  info:
```

Comment:
this field should be called _info.  
that was done because json cpp order fields by name. 
in retesteth I could move fields in json, but we have already implemented prefixed "_info" style in tests.

the comment should be copied from test filler to the final test.

Reply:
I'll fix it.